### DISCUSSION TITLE
What's the source of data? Is it legal?

### DISCUSSION THREAD

**Discussion by Dutchu** [br-g/openf1#22] [Q&A] [answered]
I was wondering if profiting out of this api is legal or should I keep it as a hobby. I'm also curious how is one in possession of such data, especially if it's a live data.

**Comment by connor-rose** [comment, marked as answer]
F1 has a public data source that can be used to create API's such as this. It's perfectly legal as far as I'm aware, the F1 data stream is publicly documented.

**Comment by OperatorHiKE** [comment]
If you're considering using an API for commercial purposes, it's important to carefully review the terms of service or use provided by the API provider to ensure that you're compliant with their requirements.
